src/main/java/org/dependencytrack/integrations/defectdojo/DefectDojoClient.java

```diff
@@ -66,7 +66,7 @@ public void uploadDependencyTrackFindings(final String token, final String engag
                 .addPart("file", inputStreamBody)
                 .addPart("engagement", new StringBody(engagementId, ContentType.MULTIPART_FORM_DATA))
                 .addPart("scan_type", new StringBody("Dependency Track Finding Packaging Format (FPF) Export", ContentType.MULTIPART_FORM_DATA))
-                .addPart("verified", new StringBody("true", ContentType.MULTIPART_FORM_DATA))
+                .addPart(Boolean.toString(verifyFindings), new StringBody("true", ContentType.MULTIPART_FORM_DATA))
```

Comment:
The key must remain "verified" and the value should be updated to `verifyFindings`.
Currently it's set as true only.